recipes-ni/ni-boot-time/ni-boot-time.bb

```diff
@@ -0,0 +1,39 @@
+SUMMARY = "Ptest for measuring and logging the boot time"
```

Comment:
Actually this is the first performance test that we've added to smoke-ptests. I've added it there because it is quick and cheap to run with the other smoke tests.

The other set of tests that upload to influxdb are the kernel-performance-tests which are run separately (due to how long they take) and are not part of smoke-ptests.

Reply:
Hmm.. OK. Could we rename this recipe to something which makes it more clear that it is a test package then? Maybe `ni-test-boott-time`?